packages/desktop-client/src/global-events.js

```diff
@@ -147,7 +147,6 @@ export function handleGlobalEvents(actions, store) {
 
   listen('start-import', () => {
     actions.closeBudgetUI();
-    actions.setAppState({ loadingText: 'Importing...' });
```

Comment:
Adding a loading text causes the entire page to re-render. Thus: after an import error occurs - we render the default state (which has no error). This is unexpected. If an error occurs - we want to show the error.

So the simple solution is to just remove this "Loading" text. We already have the loading indicator on the button itself.